docs/examples/infer-orientation-daily-peak.py

```diff
@@ -0,0 +1,90 @@
+"""
+Infer System Orientation (Azimuth & Tilt) using Daily Peak
+==========================================================
+
+Infer the azimuth and tilt of a system using its daily peak value
+"""
+
+# %%
+# Identifing and/or validating  the azimuth and tilt information for a
+# system is important, as these values must be correct for future degradation
+# and system yield analysis. This example shows how to use
+# :py:func:`pvanalytics.system.infer_orientation_daily_peak` to estimate
+# a system's azimuth and tilt, using the system's known latitude-longitude
+# coordinates and an associated AC power time series.
+
+import pvanalytics
+from pvanalytics.features.daytime import power_or_irradiance
+from pvanalytics.features.orientation import fixed_nrel
+from pvanalytics import system as sys
```

Comment:
I suggest avoiding aliasing things to the names of standard library modules.  Maybe just import the function itself instead of its host module?

Reply:
updated in commit https://github.com/pvlib/pvanalytics/pull/146/commits/7ee573577bc704d95d49e2b0b8a02d5688184c47